protocols/kad/Cargo.toml

```diff
@@ -18,9 +18,9 @@ libp2p-core = { path = "../../core" }
 log = "0.4"
 multiaddr = { path = "../../misc/multiaddr" }
 multihash = { path = "../../misc/multihash" }
-parking_lot = "0.6"
+parking_lot = "0.7"
 protobuf = "2.0.2"
-rand = "0.4.2"
+rand = "0.6.0"
```

Comment:
`0.6` might have been better here.

Reply:
`0.6` and `0.6.0` are strictly identical.